com.unity.render-pipelines.universal/ShaderLibrary/SurfaceInput.hlsl

```diff
@@ -6,9 +6,14 @@
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Packing.hlsl"
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/CommonMaterial.hlsl"
 
-TEXTURE2D(_BaseMap);            SAMPLER(sampler_BaseMap);
-TEXTURE2D(_BumpMap);            SAMPLER(sampler_BumpMap);
-TEXTURE2D(_EmissionMap);        SAMPLER(sampler_EmissionMap);
+TEXTURE2D(_BaseMap);
+SAMPLER(sampler_BaseMap);
+float4 _BaseMap_TexelSize;
```

Comment:
Should there be CBUFFER_START/END definition for these?
Also maybe sort const, textures, samples.